inference-engine/src/vpu/graph_transformer/src/stages/interpolate.cpp

```diff
@@ -95,12 +95,32 @@ void FrontEnd::parseInterpolate(const Model& model, const ie::CNNLayerPtr& _laye
             } else if (cmp(interpolateMode, "linear") || cmp(interpolateMode, "linear_onnx")) {
                 // current "Interp" supports modes "align_corners" and "asymmetric" only
                 // other "Interpolate" modes are translated to the default ones
-                const auto coordinateTransformationMode = _layer->GetParamAsString("coordinate_transformation_mode", "half_pixel");
+                const auto coordinateTransformation = _layer->GetParamAsString("coordinate_transformation_mode", "half_pixel");
+                InterpolateCoordTransMode coordinateTransformationMode = InterpolateCoordTransMode::HalfPixel;
+                InterpolateMode mode = InterpolateMode::Linear;
+
+                if (cmp(interpolateMode, "linear_onnx")) {
+                    mode = InterpolateMode::LinearOnnx;
+                }
+
+                if (cmp(coordinateTransformation, "asymmetric")) {
```

Comment:
Looks like copy-paste from another function. Could we extract this logic to a dedicated function with descriptive name?

Reply:
I guess, yes. Will take a look

Reply:
Still see this copy-paste

Reply:
yes, sure. sorry, will be fixed

Reply:
is it better now?

Reply:
@gladilov-gleb what about now?